decidim-admin/app/controllers/decidim/admin/scope_types_controller.rb

```diff
@@ -0,0 +1,81 @@
+# frozen_string_literal: true
+
+require_dependency "decidim/admin/application_controller"
+
+module Decidim
+  module Admin
+    # Controller that allows managing all scopes at the admin panel.
+    #
+    class ScopeTypesController < ApplicationController
```

Comment:
Can you catch up with #1633 here? Basically fully qualify `ApplicationController` and remove `require_dependency`.